lib/idp/constants.rb

```diff
@@ -124,6 +124,10 @@ module Vendors
 
     MOCK_IDV_APPLICANT_WITH_PHONE = MOCK_IDV_APPLICANT_WITH_SSN.merge(phone: '12025551212').freeze
 
+    MOCK_IDV_APPLICANT_WITH_ADDRESS_LINE_2 = MOCK_IDV_APPLICANT_WITH_SSN.merge(
+      address2: MOCK_IDV_APPLICANT_STATE_ID_ADDRESS[:identity_doc_address2],
```

Comment:
this reuses the existing address2, right? What if we added a new, unused address so we can make sure it's checking the correct address2?

Reply:
We're checking the presence rather than the content of address2, so I don't think we need to worry about that (if I'm interpreting your comment correctly). When I was looking closer at the test, though, I decided to remove this constant and set values in the tests themselves instead, that's in 012477e95be66b6c8e3933048ce7e5368610c6cb